### ISSUE TITLE
The Schedule plugin does not emit events when creating or updating new calendar items

### ISSUE THREAD

**Issue by ddolcimascolo** [fruux/sabre-dav#933]
The `scheduleLocalDelivery` function of the CalDAV scheduling plugin, which is responsible for updating relevant events in user's calendars as well as delivering iTip messages to Inboxes, uses raw `put` or `createFile` operations directly at the `Node` level, thus bypassing the events emitting logic.

**Comment by evert** [comment]
You're right and in your PR you also probably found out why this was done originally. In order to deliver the changes, the system basically needs to work with 'elevated privileges' because the user creating the scheduling message might not have access to the object on the target calendar.

Using `createFile` / `put` manually was a workaround. I don't yet have a good idea on how to fix this, but it's a valid bug.

**Comment by ddolcimascolo** [comment]
> in your PR you also probably found out why this was done originally

Yeah :/
